static/app/actionCreators/tags.tsx

```diff
@@ -192,3 +208,214 @@ export function fetchSpanFieldValues({
     query,
   });
 }
+
+type FetchOrganizationTagsParams = {
+  orgSlug: string;
+  // TODO: Change this to Dataset type once IssuePlatform is added
+  dataset?: string;
+  projectIds?: string[];
+  useCache?: boolean;
+};
+
+export const makeFetchOrganizationTags = ({
+  orgSlug,
+  dataset,
+  projectIds,
+  useCache = true,
+}: FetchOrganizationTagsParams): ApiQueryKey => [
+  `/organizations/${orgSlug}/tags/`,
+  {query: {dataset, useCache, project: projectIds}},
```

Comment:
This should also accept time range parameters, right? like start, end, statsPeriod